chapter.owasp.yaml

```
@@ -0,0 +1,30 @@
+leaders:
+  - name: Abdullah Munawar
+    email: abdullah.munawar@owasp.org
+  - name: Ben Pick
+    email: ben.pick@owasp.org
+    github: bennpick
+  - name: Mike McCabe
+    email: michael.mccabe@owasp.org
+    github: mccabe615
+  - name: Troy Marshal
+    email: troy.marshall@owasp.org
+    github: troymarshall
+location:
+  country: USA
+  country_code: US
+  region: North America
```

I believe it should be Virginia here, could you confirm?
```suggestion
  region: Virginia
```